src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -228,6 +227,18 @@ public static boolean anyNull(final Object... values) {
         return !allNotNull(values);
     }
 
+    /**
+     * Casts the object to the targeted class.
+     * @param value the object to cast
+     * @param castClass the class to which the object must be cast to
+     * @param <T> the type of the cast object
+     * @return the cast object, otherwise {@code null}
+     */
+    public static <T> T cast(Object value, Class<T> castClass) {
+        return (castClass != null) && castClass.isInstance(value) ? castClass.cast(value) : null;
```

Comment:
Match the style of the file: Use final where possible, no need for extra parentheses.